lib/tbot/config/template_kubernetes.go

```diff
@@ -201,6 +201,14 @@ func (t *templateKubernetes) render(
 		kubernetesClusterName: t.clusterName,
 	}
 
+	destinationDir, isDirectoryDest := destination.(*DestinationDirectory)
+	if !t.disableExecPlugin {
+		if !isDirectoryDest {
+			log.WarnContext(ctx, "Kubernetes template will be rendered without exec plugin because destination is not a directory. Explicitly set `disable_exec_plugin: true` in the output to suppress this message")
```

Comment:
Should this perhaps be emitted at `INFO` level instead, or even not at all? Some folks are pretty obsessive about warning-level errors in their logs, and this is intended behaviour.